bpf/pid.h

```diff
@@ -5,9 +5,18 @@
 #include "bpf_helpers.h"
 #include "bpf_core_read.h"
 
+#define MAX_CONCURRENT_PIDS 1000
+
 volatile const s32 current_pid = 0;
```

Comment:
We can remove these now, I guess we should just leave one flag if we are doing system wide instrumentation to skip the pid check.